Classes/Utility/LdapUtility.php

```diff
@@ -449,7 +454,12 @@ public function getDn()
      */
     public function getAttributes()
     {
-        return (@ldap_get_attributes($this->connection, $this->firstResultEntry));
+        $attributes = '';
+        if (is_resource($this->firstResultEntry)) {
+            $attributes = (@ldap_get_attributes($this->connection, $this->firstResultEntry));
```

Comment:
the parentheses around `@ldap_get_attributes` are totally useless, please remove.